snippets/csharp/Metadata.cs

```diff
@@ -0,0 +1,76 @@
+using Breez.Sdk;
+using System.Text.Json;
+
+public class ServiceStatusSnippets
+{
+    public void SetPaymentMetadata(BlockingBreezServices sdk)
+    {
+        // ANCHOR: set-payment-metadata
+        sdk.SetPaymentMetadata("target-payment-hash", "{\"myCustomValue\":true}");
+        // ANCHOR_END: set-payment-metadata
+    }
+
+    public void FilterPaymentMetadata(BlockingBreezServices sdk)
+    {
+        // ANCHOR: filter-payment-metadata
+        try
+        {
+            sdk.MetadataFilter[] metadataFilters = {
+              sdk.MetadataFilter(
+                json_path: "myCustomValue",
+                json_value: "true",
+              ),
+            };
```

Comment:
`sdk` applies just to the instance methods
```suggestion
            MetadataFilter[] metadataFilters = {
              new MetadataFilter(
                jsonPath: "myCustomValue",
                jsonValue: "true",
              ),
            };
```